services/brig/test/integration/API/User/Account.hs

```diff
@@ -773,7 +775,8 @@ testMultipleUsers brig = do
             (Just $ userDisplayName u3, Nothing)
           ]
   post
-    ( brig
+    ( apiVersion "v2"
```

Comment:
Shouldbe v3 given the finalisation of the client API v3, right?

Reply:
Updated to v3.